_scicomputing/access_aws.md

```diff
@@ -35,6 +35,16 @@ Here are some links to learn more. Keep in mind that tools provided by AWS will
 •	[AWS Pricing 101](https://aws.amazon.com/pricing/?aws-products-pricing.sort-by=item.additionalFields.productNameLowercase&aws-products-pricing.sort-order=asc&awsf.Free%20Tier%20Type=*all&awsf.tech-category=*all)
 •	[AWS Pricing Calculator](https://calculator.aws/#/?nc2=pr)
 
+## Accessing via SSO
+In 2024 Fred Hutch undertook a large migration to move all PI and Divisional AWS account to a new, more secure architecture. One of the benefits of this is integration with our Fred Hutch domain, meaning a user can now log into their AWS account using their Fred Hutch credentials. To access your account using your Fred Hutch credentials, naviagte to the [SSO Landing Page](https://d-92674cb6d7.awsapps.com/start). Once authenticated you should see all the accounts which you have access, you may then login and use the console normally. 
```

Comment:
s/which/to which/

Reply:
s/naviagte/navigate/

Reply:
https://github.com/FredHutch/wiki/pull/1036/commits/61a8f2cf2d666fa72182e07e6aee53704d681db3